packages/ingest/src/url-path-template.js

```diff
@@ -70,11 +78,17 @@ function templateReplacer(context, submatch) {
 * @returns {string} - the url path for the file
 **/
 function urlPathTemplate(pathTemplate, context) {
-  const templateRegex = /{([^}]+)}/g;
+  const templateRegex = /{([^{}]+)}/g;
```

Comment:
Why did the templateRegex change?

Reply:
The updated regex will match the inner most `{}`, e.g. match {substring(file.source, 6)} from {extractPath({substring(file.source, 6)})} 

The previous regex will match {extractPath({substring(file.source, 6)}